docs/how-to/add-model-and-api-endpoints.md

```diff
@@ -25,91 +69,189 @@ https://github.com/fyliu/peopledepot/blob/acd8898e7b0364913cc8ae3f9973dfd846aded
 
 ### Write a simple test
 
```

Comment:
Would it be helpful to add location if file & a comment?

> Open `peopledepot/app/core/tests/conftest.py`
> Note `conftest means test(s) written for structured configuration data`

Reply:
Added the file path and note about the file